python/lancedb/rerankers/linear_combination.py

```diff
@@ -0,0 +1,111 @@
+import pyarrow as pa
+
+from .base import Reranker
+
+
+class LinearCombinationReranker(Reranker):
+    """
+    Reranks the results using a linear combination of the scores from the
+    vector and FTS search. For missing scores, fill with `fill` value.
+    Parameters
+    ----------
+    weight : float, default 0.5
+        The weight to give to the vector score. Must be between 0 and 1.
+    fill : float, default 1.0
+        The score to give to results that are only in one of the two result sets.
```

Comment:
I think we should hardcode fill value. It is difficult to communicate how we're inverting one of the scores from distance to relevance but the fill=1.0 kind of represents a penalty

Reply:
Yeah I agree, hardcoding it to 1 makes sense, based on how it's being used in the code.